src/css/tailwind.css

```diff
@@ -1,3 +1,3 @@
-@tailwind base;
 @tailwind components;
 @tailwind utilities;
+@tailwind base;
```

Comment:
Why you move it? And as I remember ww decide to remove base and components imports. Or you want to remove it in another PR?